core/src/main/java/hudson/model/UpdateCenter.java

```diff
@@ -230,6 +273,74 @@ public InstallationJob getJob(Plugin plugin) {
     }
 
     /**
+     * Get the current connection status.
+     * <p>
+     * Supports a "siteId" request parameter, defaulting to "default" for the default 
+     * update site.
+     * 
+     * @return The current connection status.
+     * @since FIXME
```

Comment:
Ok, but it happens all over the place, no? `doFillTypeItems`, `doFillNameItems` etc etc (a few random ones I picked from workflow code) and most/many are not marked as `@Restricted` yet they are all web methods, no?.

But again ... I agree with you. I think all of these should be marked `@Restricted`. I'll be trying to do it from now on !